docs/examples/add-to-your-application/go/routes.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+    "fmt"
+	"log"
+	"github.com/gofiber/fiber/v2"
+)
+
+func InitApp() *fiber.App {
+    oso := initOso()
+	app := fiber.New()
```

Comment:
Seems to be some mixed tabs/spaces in this file

Reply:
thanks ran go fmt